src/compile/layout.ts

```diff
@@ -61,7 +61,7 @@ function getCellHeight(model: Model): LayoutValue {
       return model.config().cell.height;
     }
   }
-  return model.fieldDef(Y).scale.bandWidth;
+  return 21 /* config.scale.bandWidth */;
```

Comment:
If that really scares you, it should still be done in a separate PR as that involves a lot more things.  
See #1056 about facet's scale/axis.  I don't think that belongs here and it's much messier PR than here.  

If you take a look closely, it's just 21 everywhere except a few places where it should be 90.  So it is not that many numbers.  (Just two)